File: HoneydewTestProject/ReferencesABViaUsingAlias.cs
using HoneydewTestProject.A;
using C1B = HoneydewTestProject.B.C1;

namespace HoneydewTestProject
{
    public class ReferencesABViaUsingAlias
    {
        public void MethodC1A(C1 c1Params)
        {
            var x = c1Params.X;
        }

        public void MethodC1B(C1B c1Params)
        {
            var x = c1Params.X;
        }
    }
}

File: vb/HoneydewTestProjectVB/HoneydewTestProjectVB/ReferencesABViaUsingAlias.vb
Imports HoneydewTestProjectVB.A
Imports C1B = HoneydewTestProjectVB.B.C1

    Public Class ReferencesABViaUsingAlias
        Public Sub MethodC1A(ByVal c1Params As C1)
            Dim x = c1Params.X
        End Sub

        Public Sub MethodC1B(ByVal c1Params As C1B)
            Dim x = c1Params.X
        End Sub
    End Class
